dist/node/extractors/mfcc.js

```diff
@@ -28,37 +28,13 @@ exports.default = function (args) {
 			loggedMelBands[i] += filtered[i][j];
 		}
 
-		//log each coefficient
-		loggedMelBands[i] = Math.log(loggedMelBands[i]);
+		//log each coefficient unless it's 0.
+		loggedMelBands[i] = loggedMelBands[i] > 0.00001 ? Math.log(loggedMelBands[i]) : 0;
```

Comment:
Lol, that's so Maximilian-y. Almost like using ints for true/false. But here it might be so that really really small values don't make a mess in the filterbank.